File: pset6/mario/mario.py
from cs50 import get_int  # To obtain better input

# To get valid input from user
while True:
    n = get_int("Height: ")
    if (n >= 0 and n <= 23):
        break

for i in range(0, n):     # printing main loop
    for j in range(n - i - 2, -1, -1): # printing spaces
        print(" ", end = "")
    for k in range(0, i + 2):   # printing the hashes
        print("#", end="")
    print() # printing newline

# End of program

File: pset1/mario/mario.c
#include<stdio.h> //Importing input output library
#include<cs50.h> //Importing cs50 library

int main(void)
{
    int n;

    //To get valid input from user
    do
    {
        n = get_int("Height: ");
    }
    while (n < 0 || n > 23);
    for (int i = 0; i < n; i++) //printing main loop
    {
        for (int j = n - i - 2; j >= 0; j--) //printing the spaces
        {
            printf(" ");


        }
        for (int k = 0; k < i + 2; k++) //printing the hashes
        {
            printf("#");
        }

        printf("\n");   //printing newline feed or carriage return


    }
} //End of program